rpcserver.go

```diff
@@ -39,6 +40,28 @@ import (
 
 var (
 	defaultAccount uint32 = waddrmgr.DefaultAccountNum
+
+	// roMethods is a slice of method names that are considered "read-only"
+	// for authorization purposes, all lowercase.
+	roMethods = []string{
```

Comment:
If you recall our conversation, these `subscribe` methods are now using the same permissions as `list` or `get` methods as they're accessing the same data. Perhaps renaming the list from `roMethods` to something like `roPermissions` would make that more clear.